.github/workflows/ci.yml

```diff
@@ -54,7 +54,7 @@ jobs:
         uses: actions-rs/clippy-check@v1
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
-          args: --all-features --workspace --tests --examples -- -D clippy::all
+          args: --all-features --workspace --tests --examples
```

Comment:
You should still leave `-D` here, as otherwise the build won't fail with clippy lint violations.

Reply:
isn't the `env: RUSTFLAGS: -Dwarnings` taking care of that?  main.rs turns them into warnings, the env flag turns them into errors for CI.  That's what I'm expecting to happen at least.

Reply:
Ah, you're right. I completely forgot we had that, too! I'm still slightly in favor of having it in a central place for workspaces (which symbolicator will soon become), and since this is accepted practice in all our Rust repositories. 

If you still prefer to go ahead with this, please also add `warn(clippy::all)` to the other two crates in this repository, and remove from [here](https://github.com/getsentry/symbolicator/blob/b65856cde87b6e1de6472dec485c3037d0f8fd25/Makefile#L89).

Reply:
> (which symbolicator will soon become)

Noooo
 
> and since this is accepted practice in all our Rust repositories.

We already document you should add `warn(missingdocs, missing_debug_implementations)` to the crates.  I'd suggest we add `clippy::all` to that.

> If you still prefer to go ahead with this, please also add `warn(clippy::all)` to the other two crates in this repository, and remove from [here](https://github.com/getsentry/symbolicator/blob/b65856cde87b6e1de6472dec485c3037d0f8fd25/Makefile#L89).

yes, and done